lib/src/center.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:turf/bbox.dart';
+import 'package:turf/helpers.dart';
+
+/// Takes a [Feature]/[FeatureCollection] and returns the absolute center point of all features.
+Feature<Point> center<P>(GeoJSONObject geoJSON, {Map<String, dynamic>? options}) {
```

Comment:
I don't like the `options` approach here either, because we are losing the type information here too.

maybe something like this would be better:
```dart
{dynamic id, BBox? bbox, Map<String, dynamic>? properties}
```
instead of 
```dart
{Map<String, dynamic>? options}
```

Reply:
Resolved in https://github.com/dartclub/turf_dart/pull/96